### PR TITLE
Added first iteration of org data model with new org api client

### PR DESCRIPTION
**Pull request by msalte**
- Added new types for org api with a single endpoint (not yet ready)
- Added new api client
- Extended resource collection

### CODE REVIEW

File: src/http/apiClients/models/org/Location.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type Location = {
```
Comment (MartinForre) [inline src/http/apiClients/models/org/Location.ts]: Could this be a bit more specific? E.g PositionLocation?

File: src/http/apiClients/models/org/PositionInstance.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Location from './Location';
+
+type PositionInstance = {
+    obs: string;
+    percent: number;
+    appliesFrom: Date;
+    appliesTo: Date;
```
Comment (MartinForre) [inline src/http/apiClients/models/org/PositionInstance.ts]: Are we sure these will be deserialized to Date objects?

File: src/http/apiClients/models/org/Location.ts (hunk 1 above)
Comment (msalte) [reply src/http/apiClients/models/org/Location.ts]: Agreed, fixed.

File: src/http/apiClients/models/org/PositionInstance.ts (hunk 2 above)
Comment (msalte) [reply src/http/apiClients/models/org/PositionInstance.ts]: Switched to string as they will be parsed to string by default.

**Comment by MartinForre** [comment]
Should propably add the org api client to the api clients type and add it to the fusion context aswell. See https://github.com/equinor/fusion/blob/master/src/http/apiClients/index.ts

**Pull request merged**